include/revng/GraphLayout/Graphs.h

```diff
@@ -7,7 +7,7 @@
 #include <unordered_map>
 
 #include "revng/ADT/GenericGraph.h"
-#include "revng/Yield/Support/GraphLayout/Traits.h"
+#include "revng/GraphLayout/Traits.h"
```

Comment:
Yes please

Reply:
Yeah, we couldn't do this until now because it depended on the `yield::Graph`, so it would have resulted in a dependency cycle. But thanks to the #278 effort that's no longer the case.